source/docs/advanced/theme/translations.md

```diff
@@ -72,22 +73,38 @@ Please refer to [react-intl documentation](https://github.com/yahoo/react-intl/w
 
 Now that you have defined what should be translated in your application, you actually need to translate your application. This is a two-step process:
 
-1. Run the following script that will fetch all your translatable strings in your application and gather them in JSON files located in `my-module/translations/locales/[lang].json`
+1. Run the following script that will fetch all your translatable strings in your application and gather them in a JSON file located in `translations/[lang].json`
 ```sh
 npm run translate
 ```
 2. Translate the strings available in those files, and you are good to go :)
 
 Note that for some translations you won't need to change anything. This usually happens for the default language and for business phrases such as "SKU", etc.
 
-However, react-intl will warn you against those cases because it might be an actual translation that is missing. Luckily, you can opt-out of this warning for specific translations ids by adding it to the `my-module/translations/locales/whitelist_[lang].json` file.
+However, `react-intl` will warn you against those cases because it might be an actual translation that is missing. Luckily, you can opt-out of this warning for specific translations ids by adding it to the `translations/whitelist_[lang].json` file.
 
 This task can be daunting at first, but it will make your life easier in the future when you will need to know which string was not translated in a recent update.
 
+### Translations fallback
+
+With `react-intl` translations are usually grouped into a single file. In our case, we would expect them to be in `translations/whitelist_[lang].json`. But in Front-Commerce's case, we don't want you to be troubled by translations that are handled by the core.
+
+That's why Front-Commerce uses a mechanism called **translations fallbacks**. Instead of relying on a single file for translations, Front-Commerce will look out for translations in the following places and will pick the ones that exist:
```

Comment:
```suggestion
That's why Front-Commerce uses a mechanism called **translations fallbacks**. Instead of relying on a single file for translations, Front-Commerce will look out for translations in the following locations and pick the ones that exist:
```